### DISCUSSION TITLE
I have just found Fari and love it.  Questions?

### DISCUSSION THREAD

**Discussion by Stoob** [fariapp/fari-app#176] [Q&A] [answered]
So having struggled with Roll20, tried Foundry, and even the Trello sheet, I think that Fari is what I am looking for.

Thank you superb work! I have some n00b questions please.

- Is there a way to rearrange the Aspect, NPC and Index cards?  The 'sort' button doesn't seem to do anything for me I'm on chrome.  I would like to drag and drop, for example, the Bad Guys together.  What am I missing?
- For PC characters I see fate points, name, aspects.  Very cool.  Is there a way to toggle on the stress track?
- I can type in more text below a card.  Can I do this below a PC as well?  This is hard to explain so here is a screenshot.
- Will there be support for background images for a scene, or PC portraits?
![adsf](https://user-images.githubusercontent.com/1683473/102683436-ded6b180-4185-11eb-95cd-1de22e84c933.png)

**Comment by RPDeshaies** [comment, marked as answer]
Hey!

> Is there a way to rearrange the Aspect, NPC and Index cards? The 'sort' button doesn't seem to do anything for me I'm on chrome. I would like to drag and drop, for example, the Bad Guys together. What am I missing?

Sadly you just discovered a bug 😞   I'll work on a fix soon. Here's an issue for tracking purposes https://github.com/fariapp/fari/issues/177

> For PC characters I see fate points, name, aspects. Very cool. Is there a way to toggle on the stress track?

I'm not sure I understand your question here. What do you mean by toggle on the stress track ?

> I can type in more text below a card. Can I do this below a PC as well? This is hard to explain so here is a screenshot.

Not now no, what I call the "Character Cards" are kind of read only for now but that's a nice suggestion though. 

For the next version of Fari, it will be possible to add "any fields" to the Character Cards. Right now, I choose what is displayed and what isn't, but I will give users that control so that they can decide themselves what they deem important to see and what they think isn't important.

> Will there be support for background images for a scene, or PC portraits?

It's already possible! 

* just go on the web
* find an image you like
* right click + copy
* find where you want to paste it (an index card title / description  or event a field in the character sheet)
* right  + paste (or `CTRL + V` / `CMD + V`)

![image](https://user-images.githubusercontent.com/6224111/102693076-eaf25b80-41e5-11eb-8d85-5b6c4049d4d4.png)

**Comment by Stoob** [comment]
Thank you for the explanation.  These are minor improvements and already the Fari experience is excellent.  The next version sounds even better.  The 'stress track' I was imagining might look like this, I suppose? 
![stress-Pc](https://user-images.githubusercontent.com/1683473/102696903-a74c2180-41e6-11eb-87b6-fe21ffe3c85a.png)
![stress-pc2](https://user-images.githubusercontent.com/1683473/102696904-a74c2180-41e6-11eb-8b34-c2e3acfca72f.png)

**Comment by RPDeshaies** [comment]
Oh yeah so stressboxes are not visible on the Character Cards, but this is something that I might do https://github.com/fariapp/fari/milestone/10
